idea/idea-fir/src/org/jetbrains/kotlin/idea/fir/intentions/HLAddAccessorIntentions.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.fir.intentions
+
+import com.intellij.codeInsight.intention.LowPriorityAction
+import org.jetbrains.kotlin.idea.api.applicator.HLApplicator
+import org.jetbrains.kotlin.idea.api.applicator.HLApplicatorInput
+import org.jetbrains.kotlin.idea.api.applicator.applicator
+import org.jetbrains.kotlin.idea.fir.api.AbstractHLIntention
+import org.jetbrains.kotlin.idea.fir.api.applicator.HLApplicabilityRange
+import org.jetbrains.kotlin.idea.fir.api.applicator.HLApplicatorInputProvider
+import org.jetbrains.kotlin.idea.fir.api.applicator.inputProvider
+import org.jetbrains.kotlin.idea.fir.applicators.ApplicabilityRanges
+import org.jetbrains.kotlin.idea.frontend.api.symbols.KtPropertySymbol
+import org.jetbrains.kotlin.idea.intentions.AbstractAddAccessorsIntention
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.load.java.JvmAbi
+import org.jetbrains.kotlin.name.ClassId
+import org.jetbrains.kotlin.psi.KtProperty
+import org.jetbrains.kotlin.psi.psiUtil.containingClass
+import org.jetbrains.kotlin.psi.psiUtil.containingClassOrObject
+import org.jetbrains.kotlin.psi.psiUtil.hasExpectModifier
+
+abstract class HLAddAccessorIntention(private val addGetter: Boolean, private val addSetter: Boolean) :
```

Comment:
We have to do it for intentions/inspections as intention names in IJ should be distinct

Reply:
But FE1.0 intentions won't be registered in FIR plugin right?

Reply:
> But FE1.0 intentions won't be registered in FIR plugin right?

They won't be registered, right, but now we depend on FE1.0 plugin and if two inspections will have the  same fqName, we will have to relay on the classpath order. Also, it's easier to register their intentions descriptions. 

We can actually add a `HL` prefix to quickfix factories for consistency, yeah

Reply:
@darthorimar Just to be clear, are you expecting me to do that (add `HL` prefixes to quickfix factories) in this PR? I would prefer not to; it would be simpler to do it all at once in a separate change or PR.

Reply:
Sure, separate PR works